### PR TITLE
test(all): add browserstack automated unites

### PR DESCRIPTION
**Pull request by happyhj**
# Detail
* add grunt task browserstack, browserstack_runner for automated unites
* removed unused testee nam module

To use browserstack and browserstack_runner task, should set BROWSER_STACK_USERNAME, BROWSER_STACK_ACCESS_KEY environment variable.

# Prefered Reviewer
@sculove @netil 
 
# Issue
#86

### CODE REVIEW

**Comment by netil** [comment]
Great job! but, the browsers environment are described repeatedly.
Could you merge to make one env. file for all test components and specific for each component?
What do you think?

**Comment by sculove** [comment]
How about create config file on running dynamically?

**Comment by happyhj** [comment]
@netil , @sculove 
I removed every browserstack config son files, and make it generate config file dynamically when running  browserstack grunt task.
Browser support information was reference in the jsdoc @support tag.

File: Gruntfile.js [hunk 1]
```diff
@@ -7,7 +7,15 @@ module.exports = function(grunt) {
...
 	require("time-grunt")(grunt);
 	require("load-grunt-tasks")(grunt);
-	grunt.loadNpmTasks("testee");
+	
+	var env;
+	var JSON;
+	var isBrowserStack;
+	/* jshint ignore:start */
+	env = process.env;
+	JSON = JSON;
```
Comment (sculove) [inline Gruntfile.js]: How about change .jshintrc file ?
```
node:true
```

File: config/browserstack.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+		"getConfig": function(componentName){
+			var targetBrowsers = [];
+			var componentSupport = moduleList.filter(function(val){
+				return val.name === componentName;
+			}).map(function(val){
+				return val.support;
+			})[0];
+		
+			for(var browser in componentSupport) {
+				if((browser === "ie" || browser === "ios" || browser === "an" || browser === "ch" || browser === "ff") && componentSupport[browser] !== "latest") {
```
Comment (netil) [inline config/browserstack.js]: it would be better simplifying condition.
```js
/^(ie|ios|an|ch|ff)$/.test(browser)
```

Comment (happyhj) [inline config/browserstack.js]: Thank you for suggesting simplified code. I'll apply that.

File: Gruntfile.js [hunk 3]
```diff
@@ -42,6 +42,59 @@ module.exports = function(grunt) {
...
+		}
+		grunt.task.run(taskList);
+	} : function() {
+		grunt.log.oklns(
+			"no BROWSER_STACK_USERNAME, BROWSER_STACK_ACCESS_KEY env"
+		);
+	});
+
+	grunt.registerTask("browserstack_runner", isBrowserStack ? function() {	
+		var fs = require("fs");
```
Comment (mixed) [inline Gruntfile.js]: https://github.com/naver/egjs/pull/97/files#diff-35b4a816e0441e6a375cd925af50752cR80 
https://github.com/naver/egjs/pull/97/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R38
Your indent is different from default indent. Could you check your indent?

**Comment by mixed** [comment]
:+1:

**Comment by sculove** [comment]
Good job!!! LGTM

File: Gruntfile.js [hunk 4]
```diff
@@ -42,6 +42,59 @@ module.exports = function(grunt) {
...
+			});
+		}
+		grunt.task.run(taskList);
+	} : function() {
+		grunt.log.oklns(
+			"no BROWSER_STACK_USERNAME, BROWSER_STACK_ACCESS_KEY env"
+		);
+	});
+
+	grunt.registerTask("browserstack_runner", isBrowserStack ? function() {
```
Comment (sculove) [inline Gruntfile.js]: How about move browserstack* tasks to browserstack.js ?

**Comment by sculove** [comment]
LGTM

**Pull request closed without merging**